service/tests/tests.py

```diff
@@ -56,9 +56,36 @@ def test_get_massage_therapist(self):
             'end_time': '2022-05-27T16:19:52Z',
             'status': 'TBD'
         }
-        url = reverse('service:therapist_massage_list')
+        url = reverse('service:therapist_massages_list')
         self.client.force_login(user=self.user_1)
         response = self.client.get(url)
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         self.assertEqual(len(response.data), 1)
         self.assertDictEqual(dict(response.data[0]), expected_data)
+
+
+class ScheduleTest(APITestCase):
+    fixtures = ['service/tests/schedule_fixtures.json', ]
+
+    def setUp(self) -> None:
```

Comment:
If you use types, please use it for all function, or do not use at all

Reply:
Done